shardingsphere-features/shardingsphere-encrypt/shardingsphere-encrypt-core/src/main/java/org/apache/shardingsphere/encrypt/rewrite/condition/EncryptConditionEngine.java

```diff
@@ -149,6 +153,17 @@ private Optional<EncryptCondition> createCompareEncryptCondition(final String ta
                 : Optional.empty();
     }
     
+    private boolean nullOnRight(final ExpressionSegment expression) {
```

Comment:
Can we change this method name to a more meaningful one?

Reply:
OK, changed to `isNullOnTheRightOfExpressionSegment`